### PR TITLE
[wip] Timescale tests

### PR DESCRIPTION
**Pull request by shivam-kapila**
# Description
Tests for timescale_listenstore.py

### CODE REVIEW

File: listenbrainz/listenstore/tests/test_timescalelistenstore.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import logging
+
+from listenbrainz.db.testing import DatabaseTestCase
+from listenbrainz.db import timescale as ts
+from listenbrainz import config
+from listenbrainz.listenstore.tests.util import create_test_data_for_influxlistenstore, generate_data
+from listenbrainz.webserver.timescale_connection import init_ts_connection
+
+
+TIMESCALE_SQL_DIR = "/code/listenbrainz/admin/timescale"
```
Comment (paramsingh) [inline listenbrainz/listenstore/tests/test_timescalelistenstore.py]: This needs to be a relative path. See this file for examples: https://github.com/metabrainz/listenbrainz-server/blob/52705348/listenbrainz/db/__init__.py#L1

File: pytest.ini [hunk 2]
```diff
@@ -1,3 +1,5 @@
 [pytest]
 norecursedirs=listenbrainz/tests/integration listenbrainz_spark
 addopts = --cov=listenbrainz
+
+testpaths = listenbrainz/listenstore/tests/test_timescalelistenstore.py
```
Comment (paramsingh) [inline pytest.ini]: It should test this path automatically, not sure why we need this line

File: listenbrainz/listenstore/timescale_listenstore.py [hunk 3]
```diff
@@ -47,8 +47,14 @@ class InfluxListenStore(ListenStore):
     USER_LISTEN_COUNT_CACHE_TIME = 10 * 60  # in seconds. 15 minutes
 
     def __init__(self, conf, logger):
+<<<<<<< HEAD
```
Comment (paramsingh) [inline listenbrainz/listenstore/timescale_listenstore.py]: Merge conflict

File: listenbrainz/listenstore/tests/test_timescalelistenstore.py (hunk 1 above)
Comment (shivam-kapila) [reply listenbrainz/listenstore/tests/test_timescalelistenstore.py]: Yeah. I will fix that. I was just trying to get it started.

File: pytest.ini (hunk 2 above)
Comment (shivam-kapila) [reply pytest.ini]: I am running tests in only 1 file

File: listenbrainz/listenstore/timescale_listenstore.py (hunk 3 above)
Comment (shivam-kapila) [reply listenbrainz/listenstore/timescale_listenstore.py]: Removed

**Comment by mayhem** [comment]
This was merged manually. We should open a new PR for more tests and for a more thougrough review of the tests.

**Pull request closed without merging**